src/core/components/inputs/FieldWrapper.styled.ts

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components/native';
+import {indents} from "@constants/Typography";
+
+interface Props {
+    indent: keyof typeof indents;
+}
+
+export const StyledFieldWrapper = styled.View((props: Props) => ({
+    marginTop: indents[props.indent]
+}))
```

Comment:
For example, this is redundant. You're trying to predict the future and something that you'll need later on, but it's usually not needed. KISS and YAGNI.